### PR TITLE
docs: add update_service.md

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
This PR adds the update_service.md document which describes the steps to update an existing service within the project.
This document will become useful as we start to rely on the individual service teams for updating their own services within the SDK projects.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

File: update_service.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+go test
+```
+
+The unit tests should run clean.  If not, then any test failures should be diagnosed and resolved
+before proceeding.
+
+
+### Inspect new generated SDK code
+Next, it is recommended that you inspect the differences in the new and previous generated code to
+get an overall view of the changes caused by the re-generation step.   The changes that you see in the
```
Comment (pyrooka) [inline update_service.md:100]:
```suggestion
get an overall view of the changes caused by the re-generation step. The changes that you see in the
```

**Review by pyrooka — APPROVED** [review]
I only suggest a typo change, but overall it looks really good!

**Review by dpopp07 — APPROVED** [review]
Looks good! 👍

**Pull request merged**